### ISSUE TITLE
Remove shiftit recipe

### ISSUE THREAD

**Issue by wendorf** [pivotal-sprout/sprout-osx-apps#269]
Onsi's ShiftIt fork has diverged quite a bit from the current main release of ShiftIt, which is available as a homebrew cask and does not require xquartz. Thoughts?

**Comment by hiremaga** [comment]
All the projects I've been on have switched to Spectacle. I think it's a
worthy replacement for ShiftIt.

In fact, the intent when started moving toward using homebrew casks was to
deprecate this entire cookbook.

**Comment by aramprice** [comment]
I agree with dropping the recipe in favor of something/anything cask-able.

**Comment by aramprice** [comment]
I think we should archive this project. I recommend closing this issue in favor of https://github.com/pivotal-sprout/sprout-osx-apps/issues/271
